plugins/inputs/win_services/win_services.go

```diff
@@ -0,0 +1,172 @@
+// +build windows
+
+package win_services
+
+import (
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+var sampleConfig = `
+  ## Names of the services to monitor. Leave empty to monitor all the available services on the host
+  service_names = [
+    "LanmanServer",
+    "TermService",
+  ]
+`
+
+var description = "Input plugin to report Windows services info."
+
+type Win_Services struct {
+	ServiceNames []string `toml:"service_names"`
+}
+
+type ServiceInfo struct {
+	ServiceName string
+	DisplayName string
+	State       int
+	StartUpMode int
+	Error       error
+}
+
+var ServiceStatesMap = map[int]string{
+	0x00000001: "stopped",
+	0x00000002: "start_pending",
+	0x00000003: "stop_pending",
+	0x00000004: "running",
+	0x00000005: "continue_pending",
+	0x00000006: "pause_pending",
+	0x00000007: "paused",
+}
+
+var ServiceStartupModeMap = map[int]string{
+	0x00000000: "boot_start",
+	0x00000001: "system_start",
+	0x00000002: "auto_start",
+	0x00000003: "demand_start",
+	0x00000004: "disabled",
+}
+
+func (m *Win_Services) Description() string {
+	return description
+}
+
+func (m *Win_Services) SampleConfig() string {
+	return sampleConfig
+}
+
+func (m *Win_Services) Gather(acc telegraf.Accumulator) error {
+
+	serviceInfos, err := listServices(m.ServiceNames)
+
+	if err != nil {
+		return err
+	}
+
+	for _, service := range serviceInfos {
+		if service.Error == nil {
+			fields := make(map[string]interface{})
+			tags := make(map[string]string)
+
+			tags["display_name"] = service.DisplayName
+			tags["service_name"] = service.ServiceName
```

Comment:
Either skip the tag or omit the entire point, depending on if the field needs to be required.  For display name I would skip, since it is optional.  For service name I would omit the point.